RecoParticleFlow/Configuration/python/RecoParticleFlow_cff.py

```diff
@@ -36,7 +37,7 @@
                                  particleFlowEGammaFinal*
                                  pfParticleSelectionSequence )
 
-particleFlowLinks = cms.Sequence( particleFlow*particleFlowPtrs*particleBasedIsolationSequence)
+particleFlowLinks = cms.Sequence( particleFlow*particleFlowPtrs*chargedHadronIsolation*particleBasedIsolationSequence)
```

Comment:
semantically, it looks like we need to change this:
particleBasedIsolationSequence in 
RecoEgamma/EgammaIsolationAlgos/python/particleBasedIsoProducer_cff.py
to be renamed to egammaParticleBasedIsolationSequence and a new particleBasedIsolationSequence including that and the chargedHadronIsolation defined here